ui/models/treeStatus.js

```diff
@@ -30,7 +30,7 @@ export default class TreeStatusModel {
         Promise.resolve({
           result: {
             status: 'error',
-            message_of_the_day: 'Unable to connect to the <a href="https://mozilla-releng.net/treestatus">TreeStatus</a> API',
+            message_of_the_day: 'Unable to connect to the https://mozilla-releng.net/treestatus API',
```

Comment:
Curious why this is called 'message_of_the_day'. Sounds like it changes depending on the day :)

Reply:
It can, yeah.  If there are special conditions about the branch, it will be in this message.  I don't think it changes daily, but possibly could.  ``reason`` would have info about tree closures, if there was one.

Reply:
Thanks for answering my questions :)